model.ipynb

```diff
@@ -45,7 +55,77 @@
     "        features: ndarray\n",
     "            Flat vector encoding each amino acid (N x 1)\n",
     "        '''\n",
-    "        "
+    "    \n",
+    "    def get_fold(self, fold_idx = [0,1,2,3,4]):\n",
```

Comment:
No space when you pass on arguments (e.g., `fold_idx=[0,1,2,3,4]`)

Reply:
Not exactly sure what you mean here...

Reply:
@mikochen0107 for example, you wrote `def func(fold_idx = [0,1,2,3,4])`, instead, there should be no spaces around the equal sign since it's a function parameter: `def func(fold_idx=[0,1,2,3,4])`

Reply:
Also, we don't use fold 4. Maybe default is [0, 1, 2, 3]?